packages/react-components/src/shared/src/useMergedRefs.ts

```diff
@@ -1,7 +1,7 @@
 import { isFunction, isNil } from "lodash";
-import { useCallback } from "react";
+import { ForwardedRef, useCallback } from "react";
```

Comment:
I am getting confused... So there are RefObject, MutabableRefObject and ForwardedRef ?